lib/src/main/kotlin/org/walletconnect/Session.kt

```diff
@@ -73,6 +73,10 @@ interface Session {
         }
     }
 
+    interface MessageLogger {
+        fun onMessage(message: Session.Transport.Message, isOwnMessage: Boolean)
```

Comment:
Nit: Any reason why we are giving this an event-style name? Why not just `log` or `logMessage`?